traffic_ops/traffic_ops_golang/perlconfig.go

```diff
@@ -96,7 +101,7 @@ func getCDNConf(s string) (Config, error) {
 	}
 	cfg.TOURLStr = "https://127.0.0.1:" + oldPort
 	if cfg.TOURL, err = url.Parse(cfg.TOURLStr); err != nil {
-		return Config{}, fmt.Errorf("Invalid Traffic Ops URL '%v': err", cfg.TOURL, err)
+		return Config{}, fmt.Errorf("Invalid Traffic Ops URL '%v': %s", cfg.TOURL, err)
```

Comment:
I think these are backwards -- cfg.TOURL is %s, err is %v

Reply:
TOURL is not a string it is a URL struct and errors implement Stringer.